templates/admin/forms/list.html.twig

```diff
@@ -0,0 +1,154 @@
+                       data-toggle="tab">
+                        {{ "Unregistered"|translate }}
+                        &nbsp;<span class="badge">{% if unregistered|length > 0 %}{{ unregistered|length }}{% endif %}</span>
+                    </a>
+                </li>
+            </ul>
+
+            <div class="tab-content">
+                <div role="tabpanel" class="tab-pane active" id="registered">
+                    <table class="table table-striped table-condensed">
+                        <thead>
+                        <tr>
+                            <th>&nbsp;</th>
+                            <th>Name</th>
+                            <th>Status</th>
+                            <th colspan="2">&nbsp;</th>
+                            <th>{{ "Priority"|translate }}</th>
+                            <th>{{ "Category"|translate }}</th>
+                            <th>{{ "Nickname"|translate }}</th>
+                            <th>{{ "Access Control"|translate }}</th>
+                        </tr>
+                        </thead>
+                        <tbody>
+                        {% for row in registered %}
+                            <tr>
+                                <td class="text-muted"><small>{{ row.id }}</small></td>
+                                <td><strong>{{ row.name }}</strong></td>
+                                <td class="{{ row.statusColor }}">
+                                    <i class="{{ row.statusIcon }}"></i>&nbsp;
+                                    {% if row.status == "Registered" %}
+                                        {{ row.status }}
+                                    {% else %}
+                                        <a class="text-primary" href="./forms_admin.php?id={{ row.id }}&method={{ row.statusMethod }}">{{ row.status }}</a>
+                                    {% endif %}
+                                </td>
+                                <td>{{ row.packageStatus }}</td>
+                                <td class="{{ row.dbStatusClass }}">
+                                    {% if row.dbStatus == "DB Installed" %}
+                                        {{ row.dbStatus }}
+                                    {% else %}
+                                        <i class="fa fa-fw fa-database"></i>&nbsp;<a class="text-danger" href="./forms_admin.php?id={{ row.id }}&method=install_db">{{ row.dbStatus }}</a>
+                                    {% endif %}
+                                </td>
+                                <td><input type='text' size="4" class="form-control input-sm" name='priority_{{ row.id }}' value='{{ row.priority }}'></td>
+                                <td><input type='text' size="10" class="form-control input-sm" name='category_{{ row.id }}' value='{{ row.category }}'></td>
+                                <td><input type='text' size="10" class="form-control input-sm" name='nickname_{{ row.id }}' value='{{ row.nickname }}'></td>
```

Comment:
the 6 above are good examples where should be escaping with html_attr and not html